GVFS/GVFS.UnitTests/Virtualization/FileSystemCallbacksTests.cs

```diff
@@ -257,7 +257,7 @@ public void FileAndFolderCallbacksScheduleBackgroundTasks()
 
                 this.CallbackSchedulesBackgroundTask(
                     backgroundTaskRunner,
-                    (path) => fileSystemCallbacks.OnFileHardLinkCreated(path),
+                    (path) => fileSystemCallbacks.OnFileHardLinkCreated(path, string.Empty),

```

Comment:
Update this call to the version of `CallbackSchedulesBackgroundTask` that expects and action that accepts two paths?

```
        private void CallbackSchedulesBackgroundTask(
            MockBackgroundFileSystemTaskRunner backgroundTaskRunner,
            Action<string, string> callback,
            string oldPath,
            string newPath,
            FileSystemTask.OperationType operationType)
        {
```